docs/howto/metrics/proc_exposing-application-metrics.adoc

```diff
@@ -102,7 +102,7 @@ Hello from counted and timed endpoint
 . Alternatively, select `application:hello_time_mean_seconds` to see the mean time of all the invocations.
 
 Note that all metrics you created are prefixed with `application:`.
-There are other metrics, automatically exposed by {Thorntail} as the MicroProfile Metrics specification requires.
+There are other metrics, automatically exposed by {Thorntail} as the Eclipse MicroProfile Metrics specification requires.
```

Comment:
This is part of the "Metrics" guide. Could we make sure the guide uses the full name somewhere at the beginning, and then leave the "Eclipse" word out? It reads worse now.